src/AccordionPanelGroup.jsx

```diff
@@ -0,0 +1,129 @@
+import _ from 'lodash';
+import React from 'react';
+import cx from 'classnames';
+
+/**
+ * @module AccordionPanelGroup
+ */
+class AccordionPanelGroup extends React.Component {
+	constructor(props) {
+		super(props);
+		const activePanelsArr = [];
+
+		this.accordionPanels = this.props.accordionPanels.map((accordionPanel, i) => {
+			if(accordionPanel.props.isOpen) {
+				activePanelsArr.push(i);
+			}
+			return activePanelsArr;
+		});
+
+		this.state = {
+			activePanels: activePanelsArr || []
+		};
+		this._onClickTrigger = this._onClickTrigger.bind(this);
+	}
+
+	/**
+	 * Document this for PR
+	 */
+	_onClickTrigger(e, i){
+		const index = this.state.activePanels.indexOf(i);
+		const activePanelArr = this.state.activePanels;
+		const filtered = activePanelArr.filter(panel => i !== activePanelArr.indexOf(i));
+
+		if (index > -1){
+			activePanelArr.splice(index, 1);
+		} else {
+			activePanelArr.push(i);
+		}
+
+		if (!this.props.multiSelectable) {
+			_.pullAll(activePanelArr, filtered);
+		}
+
+		this.setState({
+			activePanels: activePanelArr
+		});
+
+	}
+
+	renderAccordionPanels() {
+		const toCamelCase = (str) => str.slice(0,20).replace(/(?:^\w|[A-Z]|\b\w)/g, (ltr, idx) => idx === 0 ? ltr.toLowerCase() : ltr.toUpperCase()).replace(/\s+/g, '');
+		this.accordionPanels = this.props.accordionPanels.map((accordionPanel, i) => {
+
+			return (
+				<li
+					className='list-item'
+					key={i}
+				>
```

Comment:
nitpick - should this just be on one line? also can line 51 be broken up on a few lines since its so long?